packages/server/src/script/loadRegistrationTesting.ts

```diff
@@ -0,0 +1,142 @@
+import { sampleStateInfo } from '../service/letter'
+import { contactRecords } from '../service/contact'
+import { makeClient, httpConnector } from '@tianhuil/simple-trpc/dist/client'
+import { IVbmRpc, processEnvOrThrow, wait } from '../common'
+import { promises as fs } from 'fs'
+import fetch from 'node-fetch'
+import { TimeoutError } from '@tianhuil/simple-trpc/dist/timedFetch'
+import { RpcRet, IError } from '@tianhuil/simple-trpc/dist/type'
+
+const serverUrl = processEnvOrThrow('REACT_APP_SERVER')
+const timeout = parseInt(processEnvOrThrow('REACT_APP_TIMEOUT'))
+
+export const client = makeClient<IVbmRpc>(
+  httpConnector(serverUrl, { timeout, fetch })
+)
+
+const fakeEmail = 'fake_voter@votebymail.io'
+
+const delay = async <T>(fn: () => Promise<T>, ms: number): Promise<T> => {
+  await wait(ms)
+  return fn()
+}
+
+let timeouts = 0
+let errors = 0
+
+/** Store the amount of timeouts and errors that happened in this test */
+const storeLog = async () => {
+  const file = `${__dirname}/data/loadEmailTesting.log`
```

Comment:
- If you are going to store log files, you should store json format.  Random strings like this are not helpful.
- In fact, storage is cheap so I would store more data.  Also, be sure to store the timestamp.
- Add the timestamp to the filename so that we don't overwrite old data.